components/navigation/SideBar.tsx

```diff
@@ -45,6 +47,84 @@ const MenuLink: React.FC<MenuLinkProps> = ({
   );
 };
 
+type DashboardLinkProps = {
+  siteName: string;
+  linkProps: LinkProps;
+  currentPath: string;
+  siteUUID?: string;
+  allSiteUUID?: string[];
+};
+
+const DashboardLink: React.FC<DashboardLinkProps> = ({
+  linkProps,
+  siteName,
+  currentPath,
+  siteUUID,
+  allSiteUUID,
```

Comment:
this file is getting a bit big—thoughts on extracting this out as its own component?